docs/antrea-network-policy.md

```diff
@@ -1042,13 +898,83 @@ spec:
         name: AllowToCoreDNS
 ```
 
-The above example allows all Pods from Namespace "default" to connect to all "core-dns"
+The above example allows all Pods from Namespace "default" to connect to all "kube-dns"
 Pods from Namespace "kube-system" on TCP port 53.
 
-## FQDN based filtering
+### Selecting Pods in the same Namespace with Self
+
+The `namespaces` field allows users to perform advanced matching on Namespace objects that cannot
+be done via label selectors. Currently, the `namespaces` field has only one matching strategy, `Self`.
+If set to `Self`, for each Pod affected by `appliedTo` of the policy/rule, this field selects the same
```

Comment:
I agree that the previous version had issues, but maybe this is not clear enough. What about:

> If set to `Self`, for each Pod targeted by the `appliedTo` of the policy/rule, this field will cause the rule to select endpoints in the same Namespace as that Pod.

Reply:
Done